app/src/assets/branding/credential-branding.ts

```diff
@@ -128,13 +131,15 @@ const createPersonCredentialBundle = (backgroundImageSource: string, verified =
       language: 'en',
       name: 'Person',
       issuerName: 'Service BC',
+      watermark: demo ? "NON-PRODUCTION" : undefined,
```

Comment:
Do these need to be `t('Blah.NonProduciton')`?

Reply:
no because the oca bundle is chosen by the language. The french language is handled by the french oca bundle below it:
```typescript
...
metaOverlays.push({
      captureBase: '',
      type: 'spec/overlays/meta/1.0',
      language: 'fr',
      name: 'Personne',
      issuerName: 'Service BC',
      watermark: demo ? "NON-PRODUCTION (FR)" : undefined
    })
...
```